video_xblock/static/js/studio-view/studio-edit-transcripts-manual-upload.js

```diff
@@ -132,12 +106,15 @@ function validateTranscriptFile(event, fieldName, filename, $fileUploader) {
     // Display validation error message if a transcript/caption file may not be not accepted
     if (!isValid) {
         if (fieldName === 'handout') {
-            $parentDiv = $('.file-uploader');
-            displayStatusCaptions('error', errorMessage, $parentDiv);
+            showStatus($('.status', $('.file-uploader')), 'error', errorMessage);
         } else {
             currentLiIndex = $(event.currentTarget).attr('data-li-index');
```

Comment:
> what is data-li-index?

We have dynamic list of transcripts. Each of them can be replaced. Alse we can add new. So we met a problem with identification. So we decided to store index of current li we work with in data attribute.